app/controllers/application_controller.rb

```diff
@@ -1,13 +1,8 @@
 class ApplicationController < ActionController::Base
-  before_action :set_current_user
+  before_action :allowed_parameters, if: :devise_controller?
 
-  def set_current_user
-    @current_user = current_user
-  end
-
-  helper_method :current_user
-
-  def current_user
-    User.first
+  def allowed_parameters
+    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :email, :password) }
```

Comment:
- [ ] Hello, great work, however when I try to sign up I get the below error due to unpermitted params, kindly make sure to fix this and test your application before submitting for review

![Screenshot from 2023-12-23 09-32-41](https://github.com/Rafiduzzaman/blog-app/assets/112261237/434c5357-60e8-4ea3-b063-ff9b1754bd62)